typescript-test-samples/typescript-sample-app/test/integration/integration-get-product.test.ts

```diff
@@ -68,6 +60,16 @@ describe('API Integration tests: GET Product', () => {
   let baseApiUrl :string;
 
   beforeAll(async () => {
+    /**
+     * Validate environment
+     */
+    if(!process.env.API_URL)
+      throw new Error('API_URL environment variable not set.');
+
+    mockVariable('TABLE_NAME', 'Products');
+    if(!process.env.TABLE_NAME)
```

Comment:
This guard will always fail since the environment variable is being set on the previous line, won't it?

Maybe only mock it if the person running the test doesn't already have it set in their environment, as a default?